docs/core.rst

```diff
@@ -69,6 +69,16 @@ Context
     :members:
     :exclude-members: tank
 
+Pipeline Configuration Utilities
+================================
+
+It is possible to enquiry about the location of various components of the pipeline configuration and
+the currently running version of the Toolkit Core.
+
+.. autofunction:: get_python_interpreter_for_config
```

Comment:
I had forgotten to add it way back when, but you are right, if we're deprecating it and it was never properly documented publicly, there's no point. :)